### PR TITLE
add prefix km_ in almost all kmesh bpf map

### PR DESCRIPTION
**Pull request by LiZhenCheng9527**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
a part of #1081

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by hzxuzhonghu — COMMENTED** [review]
some of the map name exceeds 16 bytes, prefer remove `map_of`

File: bpf/include/bpf_common.h [hunk 1]
```diff
@@ -53,7 +53,7 @@ struct {
     __uint(map_flags, BPF_F_NO_PREALLOC);
     __type(key, int);
     __type(value, struct sock_storage_data);
-} map_of_sock_storage SEC(".maps");
+} km_map_of_sock_storage SEC(".maps");
```
Comment (nlgwcy) [inline bpf/include/bpf_common.h:56]: More than 16 characters will be truncated. `km_sock_storage` ?

Comment (LiZhenCheng9527) [reply bpf/include/bpf_common.h]: Done

**Comment by LiZhenCheng9527** [comment]
> some of the map name exceeds 16 bytes, prefer remove `map_of`

I renamed the map in config.h

**Comment by LiZhenCheng9527** [comment]
After I change the map name in bpf/kmesh/ads/include/config.h and execute make gen, my changes are undone.
My test was to run make gen-bpf2go encountered this.

File: bpf/kmesh/workload/include/config.h [hunk 2]
```diff
@@ -14,20 +14,22 @@
 #define MAP_SIZE_OF_AUTH_TAILCALL 100000
 #define MAP_SIZE_OF_AUTH_POLICY   512
 
-// map name
+// rename map to avoid truncation when name length exceeds BPF_OBJ_NAME_LEN = 16
```
Comment (hzxuzhonghu) [inline bpf/kmesh/workload/include/config.h:17]: ? what's this

Comment (LiZhenCheng9527) [reply bpf/kmesh/workload/include/config.h:17]: Same comments as in `ads/config.h`

File: bpf/include/map_config.h [hunk 3]
```diff
@@ -0,0 +1,15 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause) */
+/* Copyright Authors of Kmesh */
+
+#ifndef _KMESH_MAP_CONFIG_H_
+#define _KMESH_MAP_CONFIG_H_
+
+// map name
+#define map_of_manager      km_manage
+#define map_of_sock_storage km_sockStorage
```
[the remark this answers is not in the document]
Comment (hzxuzhonghu) [reply bpf/include/map_config.h:9]: @LiZhenCheng9527 update it then we can merge

[the remark this answers is not in the document]
Comment (LiZhenCheng9527) [reply bpf/include/map_config.h]: Done

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**